platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/recovery/internal/MultiFileRunningHashIterator.java

```diff
@@ -33,9 +33,10 @@ public class MultiFileRunningHashIterator implements IOIterator<DetailedConsensu
     /**
      * @param iterator
      * 		the iterator that reads event stream files
+     * @throws NullPointerException in case {@code iterator} parameter is {@code null}
      */
     public MultiFileRunningHashIterator(final EventStreamMultiFileIterator iterator) {
-        CommonUtils.throwArgNull(iterator, "iterator");
+        Objects.requireNonNull(iterator, "iterator must not be null");
         this.iterator = iterator;
```

Comment:
Could combine these two lines:
```suggestion
        this.iterator = Objects.requireNonNull(iterator, "iterator must not be null");
```

Reply:
Done